tests/common/mod.rs

```diff
@@ -42,6 +42,6 @@ pub static USER_AUTH: Lazy<Authentication> = Lazy::new(|| {
 	user_auth
 });
 
-pub fn api_client() -> AuthenticatedVRC {
-	AuthenticatedVRC::new(USER_AGENT.to_owned(), USER_AUTH.clone()).unwrap()
+pub fn api_client() -> Result<AuthenticatedVRC> {
+	AuthenticatedVRC::new(USER_AGENT.to_owned(), USER_AUTH.clone())?
 }
```

Comment:
Not valid rust, should have Result error type and remove the question mark.

Reply:
I don't know how I missed that